rise/static/main.js

```diff
@@ -583,6 +579,66 @@ function reveal_cell_index(notebook) {
     return result;
 }
 
+function registerHelperActions() {
+
+    function init_metadata_slideshow(optional_cell) {
+      // use selected cell if not specified
+      var cell = optional_cell || Jupyter.notebook.get_selected_cell();
+      let metadata = cell.metadata;
+      if (metadata.slideshow === undefined)
+        metadata.slideshow = {};
+      return metadata.slideshow;
+    }
+    
+    // accessing Jupyter.actions directly results in a warning message
+    // https://github.com/jupyter/notebook/issues/2401
+    let actions = Jupyter.notebook.keyboard_manager.actions;
+    actions.register(
+      {
+        help: '(un)set current cell as a Slide cell',
+        handler: function() {
+          let slideshow = init_metadata_slideshow();
+          slideshow.slide_type = (slideshow.slide_type == 'slide') ? '' : 'slide';
+	  Jupyter.CellToolbar.rebuild_all();
+        }
+      },
+      "toggle-slide", "RISE");
+
+    actions.register(
+      {
+        help: '(un)set current cell as a Fragment cell',
+        handler: function() {
+          let slideshow = init_metadata_slideshow();
+          slideshow.slide_type = (slideshow.slide_type == 'fragment') ? '' : 'fragment';
+	  Jupyter.CellToolbar.rebuild_all();
+        }
+      },
+      "toggle-fragment", "RISE");
```

Comment:
You have `slide` and `fragment` here... I think we should add the other options as well: subslide, notes and skip.

Reply:
oh yes, maybe; it would even be possible to factorize all 5, and make this shorter;
I was an a tight schedule at the time though, and am not too available right now either to test this (plus: I  just reinstalled my mac and am still missing a lot of the basic env..)

I can do this in a separate PR if you agree

Reply:
> I can do this in a separate PR if you agree

Yes, of course!!

Reply:
I have a branch where I added the subslide action, but I would interested to see some refactoring of this piece so we can offer shortcuts for the rest of the celltoobar options without repeating code.